source/lac/trilinos_vector.cc

```diff
@@ -812,6 +799,7 @@ namespace TrilinosWrappers
       std::swap(last_action, v.last_action);
       std::swap(compressed, v.compressed);
       std::swap(vector, v.vector);
```

Comment:
There are a few other members that are not swapped here, like `nonlocal_vector` and `owned_elements`. Should they be here?